pdfarranger/pdfarranger.py

```diff
@@ -119,6 +127,14 @@
 from .config import Config
 from .core import Sides, _img_to_pdf
 
+def check_gtk_schema_exists():
+    schemas = subprocess.run(["gsettings", "list-recursively"],
+        stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+        check=True,
+        text=True)
+    return 'org.gtk.Settings.ColorChooser' in schemas.stdout
+if not check_gtk_schema_exists():
+    raise Exception('Found no schema files. You may need to set GSETTINGS_SCHEMA_DIR.')
```

Comment:
I totally understand that you added this to help/guide users with the same issue. However we don't want it to break stuff with tests that would be fine without therefore we should either:

- Run it only on darwin if it is a common enough problem here
- Make it a warning only. On windows where it is not supposed to work ever we still shouldn't run it because spamming warnings that are unfixable and not helpful are still a bad idea imo.

Feel free to do a combination of both.